src/models/SCLedger.ts

```diff
@@ -0,0 +1,3 @@
+export default interface SCLedger {
+  [address: string]: { original: string; derivative: string }
```

Comment:
`address` is `originalAddress`? Should we rename it?

Reply:
fixed, ty